src/store/getters.ts

```diff
@@ -187,23 +193,28 @@ export default {
   },
 
   /**
-   * Is True if business is in Limited Restoration state, ie
-   * the last filing to change state was a Restoration filing.
+   * True if the business is in limited restoration, ie, the state filing is a
+   * Limited Restoration filing or Limited Restoration Extension filing.
    */
-  isEntityInLimitedRestoration (_state: StateIF, _getters, rootState): boolean {
+  isInLimitedRestoration (_state: StateIF, _getters: any, rootState: any): boolean {
     const stateFiling = rootState.stateFiling // may be null
 
-    return (stateFiling?.hasOwnProperty(FilingTypes.RESTORATION))
+    return (
+      !!stateFiling && (
+        EnumUtilities.isTypeRestorationLimited(stateFiling) ||
+        EnumUtilities.isTypeRestorationExtension(stateFiling)
+      )
+    )
```

Comment:
Similar to my other comment, is it possible to access `stateFiling` via `state` and not `rootState`?

Reply:
Yes, fixed all 3.